modules/tinymce/src/core/test/ts/browser/FormatterRemoveTest.ts

```diff
@@ -306,7 +306,8 @@ describe('browser.tinymce.core.FormatterRemoveTest', () => {
     const editor = hook.editor();
     editor.setContent('<p><em><b>abc</b></em></p>');
     editor.formatter.register('format', { inline: 'b' });
-    LegacyUnit.setSelection(editor, 'b', 3, 'b', 3);
+    // Replaced LegacyUnit with TinySelections due to legacy unit producing a different result
```

Comment:
Drop these comments since if this breaks things then we go back and review the PR so see why that was done. The source wouldn't need to mention this. We are going to migrate away from LegacyUnit for other reasons overtime so when that happens this comment would not make any sense anymore.